frontend/src/pages/dashboard/Dashboard.tsx

```diff
@@ -46,34 +45,22 @@ export default function Dashboard(props: Props) {
 	const [history, setHistory] = useState<number[] | undefined>();
 	const [activeQuestionName, setActiveQuestionName] = useState<string | undefined>();
```

Comment:
what is 'setStats' and what is 'setHistory' excactly, why is this value never used?

Reply:
because process at the backend has not been quick enough..